lib/v-click-outside.js

```diff
@@ -71,19 +71,23 @@ function bind(el, { value }) {
   const instance = createInstance({ el, events, handler, middleware })
 
   instance.eventHandlers.forEach(({ event, handler }) =>
-    setTimeout(document.addEventListener, 0, event, handler),
+    setTimeout(() => document.addEventListener(event, handler), 0),
   )
   directive.instances.push(instance)
 }
 
-function update(el, { value }) {
+function update(el, { value, oldValue }) {
   const { events, handler, middleware, isActive } = processDirectiveArguments(value)
 
   if (!isActive) {
     removeInstance(el)
     return
   }
 
+  if (JSON.stringify(value) === JSON.stringify(oldValue)) {
```

Comment:
Should we move this to be the first thing to be executed in the `update` method? Above `isActive` check and `processDirectiveArguments`?

Reply:
I think you are right. I modified that. Thank you!